api/base/views.py

```diff
@@ -6,57 +6,196 @@
+    You can filter on multiple fields, or the same field in different ways, by &-ing the query parameters together.
+
+        /users/?filter[fullname]=lise&filter[family_name]=mei
+
+    ###Pagination
+
+    All collection endpoints respond to the `page` query parameter behavior as described in the [JSON-API pagination
+    spec](http://jsonapi.org/format/1.0/#crud).
+
+    ###Formatting POST/PUT/PATCH request bodies
+
+    The OSF API follows the JSON-API spec for (create and update requests)[http://jsonapi.org/format/#crud].  This means
+    all request bodies must be wrapped with some metadata.  Each request body must be an object with a `data` key
+    containing at least a `type` member.  The value of the `type` member must agree with the `type` of the entitys
+    represented by the endpoint.  If not, a 409 Conflict will be returned.  The request should also contain an
+    `attributes` member with an object containing the key-value pairs to be created/updated.  PUT/PATCH requests must
+    also have an `id` key that matches the id part of the endpoint.  If the `id` key does not match the id path part, a
+    409 Conflict error will be returned.
+
+    ##Responses
+
+    ###Entities
+
+    An entity is a single resource that has been retreived from the API, usually from an endpoint with the entity's id
+    as the final path part.  A successful response from an entity request will be a JSON object with a top level `data`
+    key pointing to a sub-object with the following members:
+
+    + `id`
+
+    The identifier for the entity.  This MUST be included with [PUT and PATCH
+    requests](#formatting-post-put-patch-request-bodies).
+
+    + `type`
+
+    The type identifier of this entity.  This MUST be included with [all create/update
+    requests](#formatting-post-put-patch-request-bodies).
+
+    + `attributes`
+
+    The properties of the entity.  Names, descriptions, etc.
+
+    + `relationships`
+
+    Relationships are urls to other entities or collections that have a relationship to the entity. For example, the
+    node entity provides a `contributors` relationship that points to the endpoint to retreive all contributors to that
+    node.  It is reccommended to use these links rather than to id-filter of general collection endpoints.  They'll be
+    faster, easier, and less error-prone.
+
+    + `links`
+
+    Links are urls to alternative representations of the entity or actions that may be performed on the entity.  Most
+    entities will provide a `self` link that is the canonical endpoint for the entity where update and delete requests
+    should be sent.  In-depth documentation of actions is available by navigating to the `self` link in the Browsable
+    API.  Most entities will also provide an `html` link that directs to the entity's page on the [OSF](http://osf.io/).
+
+    ###Collections
+
+    Collection endpoints return a list of entities and an additional data structure with pagination links, such as
```

Comment:
It is. My "collections" are the nodes used by the project organizer to collect nodes together. There will be a collections collection when that goes live, unless we rename it again. It is less than ideal.